src/main/groovy/edu/wpi/first/gradlerio/frcvision/Pi.groovy

```diff
@@ -0,0 +1,107 @@
+package edu.wpi.first.gradlerio.frcvision
+
+import groovy.transform.CompileStatic
+import jaci.gradle.deploy.context.DeployContext
+import javax.inject.Inject
+import jaci.gradle.deploy.target.location.SshDeployLocation
+import org.apache.log4j.Logger
+import org.gradle.api.Project
+import edu.wpi.first.gradlerio.frc.InvalidImageException
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+@CompileStatic
+class Pi extends FRCVisionCompatibleTarget {
+
+    private Logger log;
+
+    @Inject
+    Pi(String name, Project project) {
+        super(name, project)
+        log = Logger.getLogger(this.toString())
+
+        this.directory = '/home/pi'
+
+        this.maxChannels = 4
+
+        this.onlyIf = { DeployContext ctx ->
+            verifyOnlyIf(ctx)
+        }
+    }
+
+    void setTeam(int team) {
+        setAddresses(
+            "raspberrypi.local".toString(),                 // Default mDNS

```

Comment:
We probably shouldn't have this one, as the frcvision image doesn't use it... this is the default for "normal" Pi's that won't pass the image check anyway.